datafusion/common/src/file_options/parquet_writer.rs

```diff
@@ -242,6 +246,31 @@ impl ParquetOptions {
             skip_arrow_metadata: _,
         } = self;
 
+        /*
+        pub struct FileEncryptionProperties {
+    encrypt_footer: bool,
+    footer_key: Vec<u8>,
+    column_keys: Option<HashMap<Vec<u8>, Vec<u8>>>,
+    aad_prefix: Option<Vec<u8>>,
+
+
+    config_namespace! {
+    pub struct ConfigFileEncryptionProperties {
+        encrypt_footer: bool, default = false
+        footer_key: String, default = String::new()
+        column_keys: String, default = String::new()
+        aad_prefix: String, default = String::new()
+    }
+}
+}
+         */
+        let fep: Option<FileEncryptionProperties> =
+            match file_encryption_properties {
+                Some(fe) =>
+                    Some(fe.clone().into()),
```

Comment:
Into does a lot of work here. Converting string properties into the underlying parqet EncryptionProperties.